README.md

```diff
@@ -52,6 +52,8 @@ A local webserver is really helpful for running a version of the site on the sam
 
 Permalink buttons (in the top right hand corner of the main dashboards) will not work when running the site on local servers. This is because they are shortened with bit.ly, which doesn't allow local links.
 
+Alternatively, we can run on [npm web server](https://www.npmjs.com/package/http-server) using: `npm install http-server -g`
```

Comment:
I would prefer this moved up closer to the instructions for the python web server, and presented in a similar way.